receptorctl/.gitignore

```diff
@@ -73,7 +73,7 @@ target/
 .ipynb_checkpoints
 
 # pyenv
-.python-version
+#.python-version
```

Comment:
I don't recommend doing this as it's often used to set local virtualenvs. Not everybody is going to have Python of that specific version or could have their own content written by the pyenv commands (which may include a list of versions and virtualenvs). It's quite disruptive for people using pyenv as their workflow tool.

Reply:
Thanks for taking a look @webknjaz I had the feeling this might not be the best idea with potentially unintended side effects. With the goal of having the same Python version to pip-compile requirements with automated bumps in PRs as well as with the noxfile, is the best way likely a workflow that runs the nox session and then submits a PR? Or could there be another way to configure the version that dependabot uses?

Reply:
Thanks, @webknjaz ! Unfortunately I can't access the logs either (limited privileges in this repo). But I do see that happening in the community-website repo. I'll change that header now.